dev-docs/bidders/smn.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: SMN
+description: Prebid SMN Bidder Adaptor
+pbjs: true
+biddercode: smn
+aliasCode: admixer
+media_types: video
+gdpr_supported: true
+usp_supported: true
```

Comment:
There are many more pieces of metadata Prebid publishers have asked for. Please review the instructions at https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter. e.g. you're missing:
- gvl_id (required if you claim to support GDPR)
- fpd_supported
- schain_supported
- coppa_supported
- deals_supported
- ortb_blocking_supported
- safeframes_ok